src/index.js

```diff
@@ -220,13 +221,18 @@ export class Helmet extends Component {
   }
 
   render() {
-    const { children, helmetData, ...props } = this.props;
+    const { children, ...props } = this.props;
     let newProps = { ...props };
+    let { helmetData } = props;
 
     if (children) {
       newProps = this.mapChildrenToProps(children, newProps);
     }
 
+    if (helmetData && !(helmetData instanceof HelmetData)) {
```

Comment:
On the client, `helmetData` is a serialized object, so it doesn't include any of the `setHelmet`, `get`, `add`, and `remove` methods. But it _does_ contain the other data. So we just need to create a new HelmetData instance if it isn't already created. The underlying objects remain the same references in memory, so even though `HelmetData` is a different instance, everything works just fine! See the tests. Maybe I could create a separate test that validates the inverse?